scrapi/consumers/calpoly/README.md

```diff
@@ -0,0 +1,8 @@
+CalPoly
+=======
+
+Consumer for the Digital Commons @ CalPoly API for the SHARE project
```

Comment:
It just doesnt really make sense to have multiple readmes in a single repo.
But if its just a readme for a single module it belongs as a docstring in the module.